cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java

```diff
@@ -523,10 +523,11 @@ public ClusterModel clusterModel(long from,
       }
 
       // Populate snapshots for the cluster model.
+      Set<Integer> deadBrokers = MonitorUtils.deadBrokersWithReplicas(cluster);
```

Comment:
`MonitorUtils#deadBrokersWithReplicas(Cluster)` excludes dead brokers with 0 replicas. Shouldn't we include all dead brokers?